galsim/phase_psf.py

```diff
@@ -1905,7 +1905,7 @@ def __repr__(self):
             s += ", _force_stepk=%r" % self._force_stepk
         if self._force_maxk != 0.:
             s += ", _force_maxk=%r" % self._force_maxk
-        if self._ii_pad_factor != 4.:
+        if self._ii_pad_factor != 1.5:
```

Comment:
I was wondering if we should have these defaults controlled by a named variable (e.g., `default_ii_pad_factor`, `default_oversampling`, etc.) so that we don't have to have lines of code that depend explicitly on their values?

Reply:
Yes.  Good idea.  I'm usually usually in favor of avoiding magic numbers in code.  Better to give them a name at least.  So thanks for calling out this case where we didn't.